tests/settings.py

```diff
@@ -1,7 +1,7 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import os
 
-BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Was this change required to make the tests work? Can you explain?

Reply:
see 8c45306 : `__file__` is only absolute in python3, so it was never defined right in python2. This had no consequences before because this settings wasn't used throughout the tests but now it is.